### ISSUE TITLE
Downloading an update VS Code when checking for an update

### ISSUE THREAD

**Issue by AlexDev2020** [microsoft/vscode#119307]
- VS Code Version: 1.54.3
- OS Version: Windows 10 x64

When checking for an update, the new version is downloaded. Personally, I expect that when I click on the "Check for update" button, a check for the existence of a new version occurs and if there is one, then this is reported to the user. When I click on the "Install update" button, then the download and installation of the update should already take place.

Steps to Reproduce:

1. Install not the latest version vs code
2. Click "Chech for Updates"

Actual behavior:
Vs code finds update and downloads it.

Expected behavior:
Vs code finds the update and informs the user about it. The update does not download.

Does this issue occur when all extensions are disabled?: Yes

**Comment by joaomoreno** [comment]
Sorry about this, but we have revisited this story many times and we don't think this is worth the hassle. The vast majority of the users just wants to install the update.

**Comment by AlexDev2020** [comment]
> Sorry about this, but we have revisited this story many times and we don't think this is worth the hassle. The vast majority of the users just wants to install the update.

Eh

**Comment by AlexDev2020** [comment]
@joaomoreno
Hello. I noticed one unpleasant thing with this behavior.

Due to the fact that the update is downloaded when the update is checked, the following happens. A new version was released, vs code downloaded the update, but then a new version came out and in this situation I will have to update 2 times for one place. That is, the download and installation will be performed 2 times, which is not very pleasant. This is especially annoying on insider assemblies, which appear frequently. If vs code did not download the update when checking, then we needed one download and installation.
